neo4j/src/main/java/org/apache/metamodel/neo4j/utils/ArrayColumnTypeHandler.java

```diff
@@ -0,0 +1,39 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.metamodel.neo4j.utils;
+
+import org.apache.metamodel.schema.ColumnType;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+public class ArrayColumnTypeHandler extends AbstractColumnTypeHandler {
+    @Override
+    public ColumnType getTypeFromValue(final JSONObject data, final String key) {
+        try {
+            data.getJSONArray(key);
```

Comment:
This pattern of code gets repeated in all `ColumnTypeHandler` implementations. I would create an abstract method which contains just the code within between the `try` and `catch` and implement that within each `ColumnTypeHandler` implementation and move the remaining part of the code into the abstract class, so that would contain a concrete and an abstract method in this context.